sirepo/db_upgrade.py

```diff
@@ -63,143 +66,11 @@ def _20210211_add_flash_proprietary_lib_files(qcall, force=False):
             # Add's the flash proprietary lib files (unpacks flash.tar.gz)
             sirepo.sim_data.audit_proprietary_lib_files(
                 qcall=qcall,
-                force=force,
+                force=True,
                 sim_types=set(("flash",)),
             )
 
 
-def _20210211_upgrade_runner_to_job_db(qcall):
```

Comment:
Out of curiosity, what governs (or should govern) when db upgrades are obsolete and can be removed?

Reply:
This was a judgment call. We don't have any installations using the old runner, that's for sure. The spa session probably is the only one that's a "problem", but it is a delete, not an add, so not critical if it doesn't get done. Flash is proprietary and was technically a duplicate (see the code) so seemed like only one upgrade was necessary.